spec/lib/droplet_kit/paginated_resource_spec.rb

```diff
@@ -73,4 +73,18 @@
       end
     end
   end
-end
\ No newline at end of file
+
+  describe '#==' do
```

Comment:
For the current implementation, empty array is same as array full of nils. Implementation may change, and I've added a test case for that, but I still wonder if it's an overkill.